main.js

```diff
@@ -1,39 +1,57 @@
 let easyBoard = 10;
 let mediumBoard = 15;
 let extremeBoard = 20;
+let noBombs;
```

Comment:
You don't really need these 4 variables, especially because 3 of them are constants.